recipes/galaxy-w4m-r-classfilter/build.sh

```diff
@@ -0,0 +1,17 @@
+
+#!/bin/bash
+
+# R refuses to build packages that mark themselves as
```

Comment:
Can you please remove the comments here and the first empty line. First line should be the shebang.